docs/about/changelog.rst

```diff
@@ -8,14 +8,20 @@ For older releases, visit the `GitHub releases`_.
 
 .. _Github releases: https://github.com/Sage-Bionetworks/challengeutils/releases
 
+v2.0.0
+------
+
+**Coming soon!**
```

Comment:
Changelog can be found here: https://github.com/Sage-Bionetworks/challengeutils/pull/151